commercetools/resource_type.go

```diff
@@ -470,7 +471,7 @@ func resourceTypeFieldChangeActions(oldValues []interface{}, newValues []interfa
 		newNames[i] = v["name"].(string)
 	}
 
-	if checkAttributeOrder && !reflect.DeepEqual(oldNames, newNames[:len(oldNames)]) {
+	if checkAttributeOrder && !reflect.DeepEqual(oldNames, newNames[:int(math.Min(float64(len(newNames)), float64(len(oldNames))))]) {
```

Comment:
Can we do something like https://stackoverflow.com/a/27516559 instead of casting between float and ints?